packages/manager/modules/iam/src/components/onboarding/onboarding.template.html

```diff
@@ -0,0 +1,25 @@
+<manager-on-boarding-layout data-guides=":: $ctrl.guides">
+    <onboarding-title>
+        <p data-translate="iam_onboarding_title"></p>
+    </onboarding-title>
+    <onboarding-description>
+        <p data-translate="iam_onboarding_description"></p>
+        <p data-translate="iam_onboarding_description_2"></p>
+        <a
+            class="mt-3 oui-button oui-button_primary oui-button_l text-light"
+            data-translate="iam_onboarding_cta_1"
+            href="{{ :: $ctrl.ctaCreatePolicy }}"
```

Comment:
No need to create this property
```suggestion
            ui-sref="{{:: $root.IAM.ROUTES.CREATE_POLICY }}"
```

Reply:
It's applied.